### PR TITLE
Remove gdas bump fix files

### LINKED ISSUE #1552: Remove gdas/bump fix files
**Description**

Remove the gdas bump fix files from the primary development fix file set (`$FIX_DIR/gdas/bump/20220805`)

**Tasks**

- [x] Discuss needs with global-workflow developer assigned to request.
- [x] Remove bump fix file(s) in fix sets on supported platforms (global-workflow assignee task).
- [x] Update "Fix File Management" spreadsheet (https://docs.google.com/spreadsheets/d/1BeIvcz6TO3If4YCqkUK-oz_kGS9q2wTjwLS-BBemSEY/edit?usp=sharing).
- [x] Remove `bump_ver` from `versions/fix.ver` (https://github.com/NOAA-EMC/global-workflow/blob/develop/versions/fix.ver#L10) (Done @ [a1d93de](https://github.com/KateFriedman-NOAA/global-workflow/commit/a1d93debfa1b7c3aa7c51c9d1b9ff5cb1b18cc9c)).
- [x] Remove "bump" from `sorc/link_workflow.sh` fix linking. (Done @ [1a2ad5f](https://github.com/KateFriedman-NOAA/global-workflow/commit/1a2ad5fe65a2ea3e167d2b85cc066243ae3d6eb9)).
- [ ] Update IC folder name and tree in RTD (https://github.com/NOAA-EMC/global-workflow/pull/1562).

FYI @RussTreadon-NOAA

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
**Description**

This PR removes the `gdas/bump` fix file variable and symlink setup from global-workflow. See issue #1552 for more info.

Resolves #1552 

**Type of change**

Fix file retirement/cleanup.

**Testing**

Ran `./checkout.sh -u` and `./link_workflow.sh` in branch clone to confirm no errors introduced by removal of bump fix references.

### CODE REVIEW

File: sorc/link_workflow.sh [hunk 1]
```diff
@@ -172,7 +172,7 @@ if [[ -d "${script_dir}/gdas.cd" ]]; then
   cd "${top_dir}/fix" || exit 1
     [[ ! -d gdas ]] && mkdir -p gdas
     cd gdas || exit 1
-    for gdas_sub in bump crtm fv3jedi gsibec; do
```
Comment (RussTreadon-NOAA) [inline sorc/link_workflow.sh:175]: Other UFS-DA applications use `bump`.  While EIB currently does not stage these `bump` files, I'm guessing that you eventually want to stage `bump` files in a central location as opposed to each clone maintaining a local copy.  If this is true, do you want to retain the `bump` directory (now empty) as a visual placeholder?

Comment (aerorahul) [reply sorc/link_workflow.sh:175]: I don't mind keeping an empty directory around.

**Review by aerorahul — APPROVED** [review]
looks good.

File: sorc/link_workflow.sh (hunk 1 above)
Comment (RussTreadon-NOAA) [reply sorc/link_workflow.sh:175]: Thank you @aerorahul

Comment (aerorahul) [reply sorc/link_workflow.sh:175]: Though, how does it help here?
The next few lines are linking `bump` from the central location to here.
If there is no `bump` there, it won't be here either?

Comment (aerorahul) [reply sorc/link_workflow.sh:175]: i.e. it will be a dead link.

Comment (RussTreadon-NOAA) [reply sorc/link_workflow.sh:175]: I didn't run `link_workflow.sh` to see what happens.  Dead links aren't good.   Seems we should forget my comment and move on.

**Pull request merged**